generate/genhcl/genhcl.go

```diff
@@ -24,9 +24,9 @@ import (
 	"github.com/hashicorp/hcl/v2/hclwrite"
 	"github.com/mineiros-io/terramate/config"
 	"github.com/mineiros-io/terramate/errors"
-	"github.com/mineiros-io/terramate/globals"
 	"github.com/mineiros-io/terramate/hcl"
 	"github.com/mineiros-io/terramate/hcl/ast"
+	mcty "github.com/mineiros-io/terramate/hcl/cty"
```

Comment:
really not a big deal (or a small one), but thinkin if tmcty would not be more clear

Reply:
as we discussed on a call, maybe "github.com/mineiros-io/terramate/hcl/runtime" and we would have runtime.Object etc. not sure, but using cty seems misguided since these are our own representations of types not cty (some functions help integrating, but it is not cty I think..it would even be cool to be decoupled as much as possible from cty).